src/repos/ArticleRepo.ts

```diff
@@ -36,20 +38,35 @@ const getArticlesAfterDate = async (since: string, limit = Constants.DEFAULT_LIM
   );
 };
 
+/** A function to compare the trendiness of articles.
+ * 
+ * Trendiness is computed by taking the number of total shoutouts an article
+ * has received and dividing it by the number of days since its been published.
+ *
+ */
+export const compareTrendiness = (a1: Article, a2: Article) => {
+  const presentDate = new Date().getTime();
+  const a1Score = a1.shoutouts / (presentDate - a1.date.getTime());
```

Comment:
are you guaranteed not to get a divide by 0 error

Reply:
https://github.com/cuappdev/volume-backend/pull/13#discussion_r548705618 

Not guaranteed but I did consider it. In a future PR, I am planning on standardizing trendiness to be 1-10, so that will prevent any issues with divide by 0 error as well.